Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/measure.h

```diff
@@ -820,6 +831,186 @@ centroid(const TriangleMesh& tmesh)
   return centroid(tmesh, CGAL::Polygon_mesh_processing::parameters::all_default());
 }
 
+
+/**
+  * \ingroup measure_grp
+  * identifies faces only present in `m1` and `m2` as well as the faces present
+  * in both polygon meshes. Two faces are identical if they have the same
+  * orientation and the same points.
+  *
+  * @tparam PolygonMesh1 a model of `HalfedgeListGraph` and `FaceListGraph`
+  * @tparam PolygonMesh2 a model of `HalfedgeListGraph` and `FaceListGraph`
+  * @tparam FaceOutputIterator1 model of `OutputIterator`
+     holding `boost::graph_traits<PolygonMesh1>::%face_descriptor`.
+  * @tparam FaceOutputIterator2 model of `OutputIterator`
+     holding `boost::graph_traits<PolygonMesh2>::%face_descriptor`.
+  * @tparam FacePairOutputIterator model of `OutputIterator`
+     holding `std::pair<boost::graph_traits<PolygonMesh1>::%face_descriptor,
+     boost::graph_traits<PolygonMesh2>::%face_descriptor`.
+  *
+  * @tparam NamedParameters1 a sequence of \ref bgl_namedparameters "Named Parameters"
+  * @tparam NamedParameters2 a sequence of \ref bgl_namedparameters "Named Parameters"
+  *
+  * @param m1 the first `PolygonMesh1`
+  * @param m2 the second `PolygonMesh2`
```

Comment:
```suggestion
  * @param m1 the first `PolygonMesh1`
  * @param m2 the second `PolygonMesh2`
```
`pmesh` rather, please